### ISSUE TITLE
Flow Modern target does not create types for operation arguments

### ISSUE THREAD

**Issue by erydo** [apollographql/apollo-codegen#321]
In this example:
```graphql
mutation DoFoo($someId: ID!, $someText: String, $clientMutationId: String) {
  doFoo(input: {
      someDefault: "FOO",
      id: $someId,
      text: $someText,
      clientMutationId: $clientMutationId}) {
    someResult
  }
}
```

The old `flow` target would generate a `DoFooVariables` type including `someId`, `someText`, and `clientMutationId`.

That appears to be missing from the `flow-modern` target. In `flow-modern`, only referenced mutation input type are generated (as in—if there is an additional input type referenced in the mutation), independent of the arguments needed for the operation.

This is related to:
  * (#318) _Feature request: flow types incorporating operation name and associated variables_
  * (#320) _Flow Modern does not export "input" types_

**Comment by lewisf** [comment]
Thanks for filing! I'll take a look at implementing this when I get the chance.

**Comment by erydo** [comment]
This was closed in #348
